### ISSUE TITLE
[Accessibility] - Not able to update icon images alt text content

### ISSUE THREAD

**Issue by joaooliveiramanteigas** [Adyen/adyen-web#1622]
Describe the bug:

Use case
As a customer using screen reader or/and voice over I am not able to understand the meaning or how to use these images

Description
To meet the requirements of WCAG 2.1 ([1.1.1 non-text content level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=111#non-text-content)) images must be described with a purpose. The images we use are illustrations, but we need to describe them.

Suggested solution

Both Green checkmark and Red icon:
When a field is not validated in the payment page, the icon needs a better alt description. So it would be good if besides the translation text we could also update these icons alt text trough the configuration object, for the different languages.

Image
Image displaying where the icon presides.
![image](https://user-images.githubusercontent.com/85570868/172644463-f2397a24-e4aa-445f-97cd-c606de253dc0.png)
![image](https://user-images.githubusercontent.com/85570868/172644484-582eb5e0-adc2-4dc3-a898-cfda5048fe5a.png)

Smartphone (please complete the following information):

Device: iPhone 8
OS: iOS14.5
Browser: Chrome
Additional Context
We are using the Web Components/Drop-in v5.16.0

Thank you in advance.

**Comment by m1aw** [comment]
Hello @joaooliveiramanteigas 

Could you test this again on version 5.16.1 of adyen-web? 
This icons should no longer have alt descriptior since the error / success information should be displayed to the user via the error panel identified with "aria-live" and the success by `aria-invalid=false`. I believe this solution should meet your requirements.

**Comment by joaooliveiramanteigas** [comment]
Hi @m1aw, thanks for the quick response.
I tried updating, I'm using the embed script and stylesheet method but this new version is returning 404 for the specified links
`JS_URL_TEST: https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/${ADYEN_VERSION}/adyen.js,
  CSS_URL_TEST: https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/${ADYEN_VERSION}/adyen.css,
  JS_URL: https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/${ADYEN_VERSION}/adyen.js,
  CSS_URL: https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/${ADYEN_VERSION}/adyen.css`, 
  I also couldn't find any guide on the documentation in order to update to this version.

**Comment by m1aw** [comment]
I @joaooliveiramanteigas I believe the issue has been fixed. If not feel free to open this ticket again.
